### ISSUE TITLE
Xcode build error with version 1.7.7

### ISSUE THREAD

**Issue by nstokoe** [danwilson/google-analytics-plugin#393]
This commit:

https://github.com/danwilson/google-analytics-plugin/commit/c619273f88a7f0100d09891c4e95f0ebc5f935d9#diff-e87d8f83cd3326ebbf3f352d21103649

is giving me an error in Xcode when I try to build my app:

.../Plugins/cordova-plugin-google-analytics/UniversalAnalyticsPlugin.m:81:41: Use of undeclared identifier 'parameterName'; did you mean 'parameter'?

.../Plugins/cordova-plugin-google-analytics/UniversalAnalyticsPlugin.m:81:19: Initializing 'NSString *__strong' with an expression of incompatible type 'void'

![screen shot 2017-04-03 at 1 29 45 pm](https://cloud.githubusercontent.com/assets/1596309/24622056/a207d914-1871-11e7-8afb-b20861fe76a8.png)

Locking my plugin version to 1.7.5 alleviates the issue.

**Comment by cabaird** [comment]
I am also getting an error with the newest version while trying to build iOS
I never open xcode, so I am getting the following in the terminal during the build step
```The following build commands failed:
    CompileC /Users/cbaird/Library/Developer/Xcode/DerivedData/RenoWalk_-_lite_-_Dev-aujwzzaskfrjpafvniuzemospidl/Build/Intermediates/ArchiveIntermediates/RenoWalk\ -\ lite\ -\ Dev/IntermediateBuildFilesPath/RenoWalk\ -\ lite\ -\ Dev.build/Debug-iphoneos/RenoWalk\ -\ lite\ -\ Dev.build/Objects-normal/armv7/UniversalAnalyticsPlugin.o RenoWalk\ -\ lite\ -\ Dev/Plugins/cordova-plugin-google-analytics/UniversalAnalyticsPlugin.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
Error: Error code 65 for command: xcodebuild with args: -xcconfig,/Users/cbaird/Documents/Development/THD Workspace/RenoWalk_Ionic/platforms/ios/cordova/build-debug.xcconfig,-workspace,RenoWalk - lite - Dev.xcworkspace,-scheme,RenoWalk - lite - Dev,-configuration,Debug,-destination,generic/platform=iOS,-archivePath,RenoWalk - lite - Dev.xcarchive,archive,CONFIGURATION_BUILD_DIR=/Users/cbaird/Documents/Development/THD Workspace/RenoWalk_Ionic/platforms/ios/build/device,SHARED_PRECOMPS_DIR=/Users/cbaird/Documents/Development/THD Workspace/RenoWalk_Ionic/platforms/ios/build/sharedpch```
I rolled back to version 1.7.4, which is the version I installed previously, and everything works fine.

**Comment by victorsosa** [comment]
thanks, fix is up, please update to 1.7.9

**Comment by nstokoe** [comment]
Confirmed that it is fixed with 1.7.9. Thanks for the speedy fix.

**Comment by victorsosa** [comment]
you welcome @nstokoe
